compiler/rustc_const_eval/src/interpret/eval_context.rs

```diff
@@ -569,7 +569,7 @@ impl<'mir, 'tcx: 'mir, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
     ) -> Result<T, ErrorHandled> {
         frame
             .instance
-            .try_subst_mir_and_normalize_erasing_regions(
+            .try_instantiate_mir_and_normalize_erasing_regions(
```

Comment:
FWIW I found the previous term much clearer (it's a generic function and we are substituting in concrete values)... but I guess that ship has sailed so it probably makes sense to make this consistent.

The function this is in is called `subst_from_frame_and_normalize_erasing_regions`. So do you suggest that should be renamed to `instantiate_from_frame_and_normalize_erasing_regions` or so?

Reply:
> The function this is in is called `subst_from_frame_and_normalize_erasing_regions`. So do you suggest that should be renamed to `instantiate_from_frame_and_normalize_erasing_regions` or so?

yeah, it does feel a bit verbose though :sweat_smile: :shrug: